tests/redis-trib.rb

```diff
@@ -0,0 +1,1402 @@
+#!/usr/bin/env ruby
```

Comment:
@Grokzen it means thath i should do the followding steps,right?
1. just add the original version of redis-trib.rb from reidis repo,  and commit ,in the comment, add the hash(or version) of the file
2. commit changes I made upon it